backend/src/test/scala/com/softwaremill/adopttapir/starter/FileOperation.scala

```diff
@@ -2,13 +2,20 @@ package com.softwaremill.adopttapir.starter
 
 import cats.effect.{ExitCode, IO, IOApp}
 import com.softwaremill.adopttapir.config.Config
+import com.softwaremill.adopttapir.starter.files.FilesManager
+import com.softwaremill.adopttapir.starter.formatting.ProjectFormatter
 import com.softwaremill.adopttapir.template.ProjectGenerator
 
 @deprecated("Only for development purpose")
 object FileOperation extends IOApp {
 
-  private val cfg = Config.read.starter
-  val service = new StarterService(cfg.copy(deleteTempFolder = false), new ProjectGenerator())
+  val service: StarterService = {
+    val cfg = Config.read.storageConfig.copy(deleteTempFolder = false)
+    val pg = new ProjectGenerator()
+    val fm = new FilesManager(cfg)
+    val pf = new ProjectFormatter(fm)
+    new StarterService(pg, fm, pf)
```

Comment:
nit: naming - abbreviations are harder to follow as one needs to remember what they are standing for simple `projectGenerator`, `filesManager`, `filesFormatter` would do

Reply:
I would rather leave it as it is - please notice, that all of these are used in stricly limited scope in order to create `StarterService`.

Reply:
Ok, so maybe we don't need all of them?
How about
```
val fm = new FilesManager(cfg)
new StarterService(new ProjectGenerator(), fm, new ProjectFormatter(fm))
```
?

Reply:
Updated.